src/common/transformations/src/transformations/op_conversions/convert_ti_to_sequences.cpp

```diff
@@ -238,15 +235,15 @@ ngraph::pass::ConvertTensorIteratorToLSTMSequence::ConvertTensorIteratorToLSTMSe
         // create a pattern for the TensorIterator body
         auto data = ngraph::pattern::wrap_type<ngraph::opset5::Parameter>(ngraph::pattern::rank_equals(3));
         auto pattern_1 = ngraph::pattern::wrap_type<ngraph::opset5::Constant>(ngraph::pattern::rank_equals(1));
-        auto squeeze = ngraph::pattern::wrap_type<ngraph::opset5::Reshape>({ data, pattern_1 });
+        auto squeeze = ngraph::pattern::wrap_type<ngraph::opset5::Reshape, ngraph::opset5::Squeeze>({data, pattern_1});
```

Comment:
Should we add Unsqueeze node to this pattern also?

Reply:
Unsqueeze after Cell replaces to Reshape because in case of fully dynamic data output cell shape contains only 1 dynamic dimension (second dimension is static because it can be extracted from weights).